Makefile

```diff
@@ -7,6 +7,25 @@ CERT_MANAGER_CHART_VERSION := 1.11.0
 
 VERSION_PACKAGE := github.com/akuity/kargo/internal/version
 
+IMAGE_REPO ?= kargo
+IMAGE_TAG ?= latest
```

Comment:
```suggestion
IMAGE_TAG ?= $(shell git describe --always --abbrev=7 --dirty --match=NeVeRmAtCh)
```

Reply:
This would address [my only outstanding concern](https://github.com/akuity/kargo/pull/542#issuecomment-1673583866)